api/BeforeUnloadEvent.json

```diff
@@ -86,6 +92,9 @@
               "version_added": "60"
             },
             "chrome_android": "mirror",
+            "deno": {
+              "version_added": false
+            }
```

Comment:
Looks like you've got a small syntax error:

```suggestion
            },
```

Reply:
Oops. Fixed. Sorry about that.